crates/starknet/src/starknet/mod.rs

```diff
@@ -104,7 +117,10 @@ pub struct Starknet {
 }
 
 impl Starknet {
-    pub fn new(config: &StarknetConfig) -> DevnetResult<Self> {
+    pub fn new(
+        config: &StarknetConfig,
+        transactions: Option<StarknetTransactions>,
```

Comment:
IMO loading transaction shall be done by outside entity like `DumpLoader`. `Starknet` just propagates calls to starknet_in_rust and filesystem(now) loading itself looks like not his responsibility.
DumpLoader has function `load` that returns `Starknet`.

Reply:
Now transactions are removed in the constructor but I'll call you or dm about DumpLoader.

Reply:
Sure

Reply:
This can be addressed later once you will come back from vacation https://github.com/0xSpaceShard/starknet-devnet-rs/issues/197